iesurveychecks/3-complementary-analysis-r.md

```diff
@@ -0,0 +1,18 @@
+
+# Complementary Quality Checks Analysis
+
+In addition to the real time data quality checks published on the dashboard using [the HFC dashboard](), additional checks can be done on statistical software for checks that are difficult to visualize using pivot tables or those which require a deeper dive. These checks include:  
+
+1. <b> Duplicates check </b>: Produce a list for duplicates for complete surveys (same unique ID submitted twice but with different answers - we need to check with field team which one to keep) (see [sample_checks.R](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/iesurveychecks/r/sample_checks.R))
```

Comment:
rather than referring to the whole R script, it would be better to either refer to specific lines or subsections of code or to break up the code into separate files (the latter may be desirable for code template purposes but probably more work than needed in the immediate term).

Reply:
i agree. we can do this after the webinar